app/controllers/signup.js

```diff
@@ -1,194 +1,219 @@
-    firstName: '',
-    lastName: '',
-    username: '',
-    email: '',
-    phoneNumber: '',
-    experience: '',
-    companyName: '',
-    designation: '',
-    github: '',
-    linkedIn: '',
-    twitter: '',
+    first_name: "",
+    last_name: "",
+    username: "",
+    email: "",
+    phone_number: "",
+    yoe: "",
+    company_name: "",
+    designation: "",
+    github_id: "",
+    linkedin_id: "",
+    twitter_id: "",
   };
 
   @tracked formErrors = {
-    firstName: false,
-    lastName: false,
+    first_name: false,
+    last_name: false,
     username: false,
     email: false,
-    phoneNumber: false,
-    experience: false,
-    companyName: false,
+    phone_number: false,
+    yoe: false,
+    company_name: false,
     designation: false,
-    github: false,
-    linkedIn: false,
-    twitter: false,
+    github_id: false,
+    linkedin_id: false,
+    twitter_id: false,
   };
 
   @tracked fields = [
     {
-      id: 'firstName',
-      label: 'First Name',
-      type: 'text',
-      errorMessage: 'First name is required',
+      id: "first_name",
+      label: "First Name",
+      type: "text",
+      errorMessage: "First name is required",
       required: true,
-      showError: false
+      showError: false,
     },
     {
-      id: 'lastName',
-      label: 'Last Name',
-      type: 'text',
-      errorMessage: 'Last name is required',
+      id: "last_name",
+      label: "Last Name",
+      type: "text",
+      errorMessage: "Last name is required",
       required: true,
-      showError: false
+      showError: false,
     },
     {
-      id: 'username',
-      label: 'Choose a username',
-      type: 'text',
-      errorMessage: 'Username is required',
+      id: "username",
+      label: "Choose a username",
+      type: "text",
+      errorMessage: "Username is required",
       required: true,
-      showError: false
+      showError: false,
     },
     {
-      id: 'email',
-      label: 'Email',
-      type: 'email',
-      errorMessage: 'Valid Email is required',
+      id: "email",
+      label: "Email",
+      type: "email",
+      errorMessage: "Valid Email is required",
       required: true,
       showError: false,
-      validator: this.emailValidator
+      validator: this.emailValidator,
     },
     {
-      id: 'phoneNumber',
-      label: 'Phone Number',
-      type: 'string',
-      value: '+91-',
-      errorMessage: 'Enter a valid phone number',
+      id: "phone_number",
+      label: "Phone Number",
+      type: "string",
+      value: "+91-",
+      errorMessage: "Enter a valid phone number",
       required: false,
       showError: false,
-      validator: this.phoneNumberValidator
+      validator: this.phone_numberValidator,
     },
     {
-      id: 'experience',
-      label: 'Experience',
-      type: 'number',
-      errorMessage: 'Number of experience is required',
+      id: "yoe",
+      label: "yoe",
+      type: "number",
+      errorMessage: "Number of yoe is required",
```

Comment:
A better error message would be `Number of years of experience is required`

Reply:
replaced!